extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -138,6 +140,8 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 		return fillInSlotWithDelay;
 	}
 
+	messageListener.register({dataKey: 'disableBtf', infinite: true}, disableBtfByMessage);
```

Comment:
Why is it registered twice?

Reply:
It's being cleared on each pv:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AdEngine/js/MessageListener.js#L103